### DISCUSSION TITLE
Error Parsing Post

### DISCUSSION THREAD

**Discussion by TomInVa57** [TumblThreeApp/TumblThree#306] [General]
Hey. TumblThree just downloaded and installed the latest update. I am not able to download any likes, all giving error message "error parsing post". Can you fix this please :) Thank you.

**Comment by thomas694** [comment]
Thanks for reporting a problem. We'll have a look at it.

**Comment by bloke90** [comment]
I have same issue - pls don't tell us to download zip-file and overwrite ...

**Comment by thomas694** [comment]
> Can you fix this please :) Thank you.

Welcome. 🙂 

> pls don't tell us to download zip-file and overwrite ...

You can also press on 'Yes' when it asks to update to newest version. 😉

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
